src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java

```diff
@@ -659,12 +659,6 @@ private void doPushImage(String url, Map<String, String> header, HcChunkedRespon
     private String tagTemporaryImage(ImageName name, String registry) throws DockerAccessException {
         String targetImage = name.getFullName(registry);
         if (!name.hasRegistry() && registry != null) {
-            if (hasImage(targetImage)) {
```

Comment:
@jroper : rather than removing exception thrown, can we log it maybe in warn mode? Wdyt?